### ISSUE TITLE
Replace stylistic rules with prettier plugin

### ISSUE THREAD

**Issue by fregante** [xojs/stylelint-config-xo#16]
Stylelint does a less-than-stellar job at formatting, so what do you think about adding this plugin to the config to defer this job to Prettier **by default**?

https://github.com/prettier/stylelint-prettier

**Comment by connorjs** [comment]
FYI - Stylelint 15 has deprecated stylistic rules, which means the `stylelint-prettier-plugin` has no utility v15+.

<https://stylelint.io/migration-guide/to-15>

**Comment by fregante** [comment]
> the stylelint-prettier-plugin has no utility v15+.

Doesn't v15 mean exactly the opposite? _Because_ stylelint doesn't have formatting rules, you _should_ use Prettier either separately or as a plugin. 

In fact the plugin was readied for v15 by dropping its overriding config: https://github.com/prettier/stylelint-prettier/issues/284

**Comment by fregante** [comment]
I think you're confusing stylelint-config-prettier with stylelint-**plugin**-prettier

Either way, great move by stylelint, they were worse than eslint at formatting. I hope eslint takes note and follows suit.

**Comment by connorjs** [comment]
Correct! (a) I typed without checking specifics and (b) mis-read the intent when glancing through issues. 🤦🏻‍♂️🤦🏻‍♂️

> Additionally, you may no longer need to extend [Prettier's Stylelint config](https://www.npmjs.com/package/stylelint-config-prettier) as there should be no conflicting rules:
>
> ```
> {
> - "extends": ["stylelint-config-prettier"],
>   "rules": { .. }
> }
> ```

---

> Because stylelint doesn't have formatting rules, you should use Prettier either separately or as a plugin.

👍🏻

**Comment by fregante** [comment]
Yeah the confusing part is that they called the config a "plugin" in a few places 🤷‍♂️

**Comment by sindresorhus** [comment]
I personally don't plan to use Prettier, but I'm happy to have the current config split into stylistic and non-stylistic if that would help.
